docs/source/tutorial.rst

```diff
@@ -0,0 +1,577 @@
+                                         lon: 1440)
+    Coordinates:
+      * lat                             (lat) float32 90.0 89.75 ... -89.75 -90.0
+      * lon                             (lon) float32 nan 0.25 0.5 ... 359.5 359.8
+      * new_dimension                   (new_dimension) object '1' '2'
+      * time0                           (time0) datetime64[ns] 2020-01-01 ... 202...
+    Data variables:
+        air_pressure_at_mean_sea_level  (new_dimension, time0, lat, lon) float32 ...
+    Attributes:
+        institution:  ECMWF
+        source:       Reanalysis
+        title:        ERA5 forecasts
+
+
+Similarly we can map each file to a new variable using the special ``var`` character in coo_map. Here we use the same ``regex`` function but instead map these as new variables.
+
+.. code:: 
+
+    mzz = MultiZarrToZarr(json_list,                
+        remote_protocol='s3',
+        remote_options={'anon':True},
+        coo_map = {"var":ex},
+        concat_dims=['time0'],
+        identical_dims = ['lat', 'lon']
+    )
+    
+    d = mzz.translate()
+
+    backend_args = {"consolidated": False, "storage_options": {"fo": d, "remote_protocol": "s3","remote_options": {"anon": True}}}
+    print(xr.open_dataset("reference://", engine="zarr", backend_kwargs=backend_args))
+
+
+.. parsed-literal::
+
+    <xarray.Dataset>
+    Dimensions:  (time0: 1440, lat: 721, lon: 1440)
+    Coordinates:
+      * lat      (lat) float32 90.0 89.75 89.5 89.25 ... -89.25 -89.5 -89.75 -90.0
+      * lon      (lon) float32 nan 0.25 0.5 0.75 1.0 ... 359.0 359.2 359.5 359.8
+      * time0    (time0) datetime64[ns] 2020-01-01 ... 2020-02-29T23:00:00
+    Data variables:
+        1        (time0, lat, lon) float32 ...
+        2        (time0, lat, lon) float32 ...
+    Attributes:
+        institution:  ECMWF
+        source:       Reanalysis
+        title:        ERA5 forecasts
+
+
+Another special character in ``coo_map`` is ``attr:``. This allows the user to access variables from the file attributes.
+
+.. code:: 
+
+    mzz = MultiZarrToZarr(json_list,                
+        remote_protocol='s3',
+        remote_options={'anon':True},
+        coo_map = {"var":"attr:institution"},
+        concat_dims=['time0'],
+        identical_dims = ['lat', 'lon']
+    )
+    
+    d = mzz.translate()
+
+    backend_args = {"consolidated": False, "storage_options": {"fo": d, "remote_protocol": "s3","remote_options": {"anon": True}}}
+    print(xr.open_dataset("reference://", engine="zarr", backend_kwargs=backend_args))
+
+
+.. parsed-literal::
+
+    <xarray.Dataset>
+    Dimensions:  (time0: 1440, lat: 721, lon: 1440)
+    Coordinates:
+      * lat      (lat) float32 90.0 89.75 89.5 89.25 ... -89.25 -89.5 -89.75 -90.0
+      * lon      (lon) float32 nan 0.25 0.5 0.75 1.0 ... 359.0 359.2 359.5 359.8
+      * time0    (time0) datetime64[ns] 2020-01-01 ... 2020-02-29T23:00:00
+    Data variables:
+        ECMWF    (time0, lat, lon) float32 ...
```

Comment:
but both datasets had the save value of the attribute, so is this actually what we expect?

Reply:
In this case the `json_list` contains two files of consecutive time steps and the same variable